lib/ddtrace/configuration.rb

```diff
@@ -119,6 +107,19 @@ def components(allow_initialization: true)
 
     private
 
+    # Gracefully shuts down the tracer and disposes of component references,
+    # allowing execution to start anew.
+    #
+    # In contrast with +#shutdown!+, components will be automatically
+    # reinitialized after a reset.
```

Comment:
Perhaps add here a note that should only be used for tests?

Reply:
👍